src/components/tutorials/basic-tutorial.js

```diff
@@ -90,19 +58,19 @@ class BasicTutorial extends React.Component {
-            <p>If you look at the first chunk of <code>JSON</code> in the <code>"parts"</code> list, the IA component of final score reads <code>3.75</code>. The <code>"detail"</code> makes clear that the scoring is based on the measurement set we submitted earlier (check the <code>id</code>s, they match!).</p>
+            <p>The <code>Final Score</code> has a value of <code>7.5</code>. <code>Final</code> here doesn't mean it's the final end of year score for this submission - think of this instead as the <em>current aggregate estimate</em> of your total score.</p>
+            <p>If you look at the first chunk of <code>JSON</code> in the <code>"parts"</code> list, the IA component of final score reads <code>7.5</code>. The <code>"detail"</code> makes clear that the scoring is based on the measurement set we submitted earlier (check the <code>id</code>s, they match!).</p>
             <p>The second chunk in <code>"parts"</code> is straightforward - we did not submit any ACI data, so our aggregate score has no contribution from ACI measures.</p>
             <p>That's it! We've used three API requests to (1) create a submission, (2) add measurements into a measurement set, and (3) get the score, all in minutes. Each API response gives us useful information for the next step.</p>
             <p>In general, we can think about the Submissions API as a way to have a live conversation with CMS about performance measurements. Rather than waiting months to hear back about missing information or a score, the API gives us feedback that is immediate, specific, and actionable - we can easily make another API request if necessary.</p>
             <p>What we've shown is an example of working directly with the API - typically these requests are made through a web interface or script, but the requests & responses above illustrate the kind of power and speed the Submissions API and applications built against it can provide.</p>
             <p><em>Disclaimer:</em> Scoring is subject to change, based on periodic policy updates, eligibility reviews, and technical integration developments.</p>
 
             <h3>Next steps</h3>
-            <p>Demystify how the aggregate score for a complex performance category like ACI by seeing an example of how it might be calculated. Updating and correcting performance data is also easy - avoid losing time by solving issues as they arise, rather than reacting months later.</p>
-            <a className='ds-c-button ds-c-button--primary' href='/qpp-submissions-docs/advanced-tutorial'>Start the advanced tutorial</a>
+            <p>Updating and correcting performance data is also easy - avoid losing time by solving issues as they arise, rather than reacting months later. Go to the advanced tutorial to see how you can update a measurement set that you've previously submitted.</p>
```

Comment:
update this paragraph to:

<p>Updating and correcting performance data is also easy - avoid losing time by solving issues as they arise, rather than reacting months later. Go to the next tutorial to see how you can update a measurement set that you've previously submitted.</p>